components/brave_shields/browser/brave_shields_util.cc

```diff
@@ -243,22 +244,38 @@ bool ShouldDoDebouncing(HostContentSettingsMap* map, const GURL& url) {
   return true;
 }
 
-bool ShouldDoDomainBlocking(HostContentSettingsMap* map, const GURL& url) {
+DomainBlockingType GetDomainBlockingType(HostContentSettingsMap* map,
+                                         const GURL& url) {
   // Don't block if feature is disabled
   if (!base::FeatureList::IsEnabled(brave_shields::features::kBraveDomainBlock))
-    return false;
+    return DomainBlockingType::kNone;
 
   // Don't block if Brave Shields is down (this also handles cases where
   // the URL is not HTTP(S))
   if (!brave_shields::GetBraveShieldsEnabled(map, url))
```

Comment:
This goes against the established patter of using `Get` for the raw value for display in the UI and I think that's a bit confusing

Reply:
to be clear I'm talking about `GetDomainBlockingType`, just commenting here because of the shields check

Reply:
maybe `absl::optional<DomainBlockingType> ShouldDoDominBlocking(...)` and get rid of `kNone`?